internal/protoc/compiler.go

```diff
@@ -268,7 +270,10 @@ func (c *compiler) getCmdMetas(protoSet *file.ProtoSet) (cmdMetas []*cmdMeta, re
 	}()
 	// you need a new downloader for every ProtoSet as each configuration file could
 	// have a different protoc.version value
-	downloader := c.newDownloader(protoSet.Config)
+	downloader, err := c.newDownloader(protoSet.Config)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create downloader: %v", err)
```

Comment:
nit: you could just return the error, Prototool does not do aggressive error wrapping